tools-local/init-build.proj

```diff
@@ -67,6 +69,31 @@
      <Exec Condition="'$(Platform)' == 'armel'" Command="$(ArmEnvironmentVariables) $(ProjectDir)cross/armel/tizen-build-rootfs.sh" />
   </Target>
 
+  <Target Name="BuildReferenceAssemblies" Condition="'$(OfflineBuild)' == 'true'">
+    <ItemGroup>
+      <IlSource Include="$(ReferencePackagesSourceDir)/**/*.il" />
+      <ReferenceAssemblyDirectories Include ="$(ReferencePackagesStagingDir)/**/*.nuspec" />
+    </ItemGroup>
+
+    <MakeDir Directories="$(ReferencePackagesStagingDir)%(IlSource.RecursiveDir)" Condition="!Exists('$(ReferencePackagesStagingDir)%(IlSource.RecursiveDir)')" />
```

Comment:
Is the condition necessary? I thought it just kept going if it exists.

Reply:
I'll try without.